lib/placebo/meck.ex

```diff
@@ -107,7 +107,7 @@ defmodule Placebo.Meck do
         next.(default)
 
       :get, data, next ->
-        next.(data)
+        next.([data])
```

Comment:
```suggestion
        data
        |> List.wrap()
        |> next.()
```

Reply:
Changing from a hard wrap to `List.wrap/1` means we don't need to `List.flatten/1` elsewhere. Didn't think of it until now.